app/src/main/java/com/example/apprenticeshipcrypto/network/BitsoApiService.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.apprenticeshipcrypto.network
+
+import retrofit2.Retrofit
+import retrofit2.http.GET
+
+private const val BASE_URL = "https://api.bitso.com/v3/"
+
+private val retrofit = Retrofit.Builder()
```

Comment:
If we will use Dependency Injection with Hilt, we should create a separate class that holds the API logic (creation of Retrofit instance) to be injected when needed